### ISSUE TITLE
Failing to retrieve artifact

### ISSUE THREAD

**Issue by jub77** [jk1/Gradle-License-Report#24]
Creation reports a failed retrieval because one of the dependecies contains pom with following definition of parent (org.apache.geronimo.specs:specs:1.1):

```    
    <parent>
        <artifactId>apache</artifactId>
        <groupId>
            org.apache
        </groupId>
        <version>1</version>
    </parent>
```
The groupId contains spaces and new lines and it cause unsuccessful resolving of artifacts. Probably some trimming of the value missing

```
Failed to retrieve artifacts for [group:
            org.apache
        , name:apache, version:1, ext:pom]
org.gradle.api.artifacts.ResolveException: Could not resolve all dependencies for configuration ':dependencyLicenseReport678'.
	at org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration.getFailure(DefaultLenientConfiguration.java:144)
	at org.gradle.api.internal.artifacts.ivyservice.DefaultLenientConfiguration.rethrowFailure(DefaultLenientConfiguration.java:157)
	at org.gradle.api.internal.artifacts.ivyservice.DefaultResolvedConfiguration.rethrowFailure(DefaultResolvedConfiguration.java:43)
	at org.gradle.api.internal.artifacts.ivyservice.DefaultResolvedConfiguration.getResolvedArtifacts(DefaultResolvedConfiguration.java:76)
	at org.gradle.api.internal.artifacts.ivyservice.ErrorHandlingConfigurationResolver$ErrorHandlingResolvedConfiguration.getResolvedArtifacts(ErrorHandlingConfigurationResolver.java:296)
	at sun.reflect.GeneratedMethodAccessor153.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:93)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:325)
	at org.codehaus.groovy.runtime.metaclass.MethodMetaProperty$GetBeanMethodMetaProperty.getProperty(MethodMetaProperty.java:76)
	at org.codehaus.groovy.runtime.callsite.GetEffectivePojoPropertySite.getProperty(GetEffectivePojoPropertySite.java:64)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callGetProperty(AbstractCallSite.java:296)
	at com.github.jk1.license.util.CachingArtifactResolver.doResolveArtifact(CachingArtifactResolver.groovy:43)
```

**Comment by jk1** [comment]
It looks like I managed to create a report with that dependency without issues. What gradle and plugin versions you're using?

**Comment by jub77** [comment]
I'm using gradle 4.1. Here is the part of the build script which failes:

```
plugins {
  id 'com.github.jk1.dependency-license-report' version '0.3.12'
  id 'java'
}

repositories {
  mavenCentral()
}

dependencies {
  compile "org.apache.geronimo.specs:geronimo-jta_1.0.1B_spec:1.0.1"
}
```

**Comment by jk1** [comment]
I see now, thanks. 
Please find it fixed in 0.3.13.
